content/en/blog/2022/metrics-announcement.md

```diff
@@ -0,0 +1,34 @@
+    - Anyone who wants to capture metrics from their hosts (Linux VMs, Windows, VMs, Kubernetes, etc.) or pre-packaged applications (databases, message queues, etc.).
+    - Anyone who wants to capture metrics from existing sources like OTLP (from OpenTelemetry SDKs, pre-packaged applications, etc.), Prometheus, or others.
+    - Anyone who wants to process / modify metrics and metric metadata captured from these sources.
+    - Anyone who wants to convert metrics from one format to another. For example, the Collector can capture metrics from a mix of OTLP and Prometheus sources, and then send all of these to a single destination using OTLP (with the standard OpenTelemetry semantic conventions), Prometheus, or any other exporter.
+- Full OpenTelemetry Protocol (OTLP) support for efficiently serializing and transmitting metrics between systems.
+- A metrics section of the specification, which defines different types of metrics, their shapes, how to process them, and semantic conventions. This is primarily used by OpenTelemetry contributors but also provides guidance to OpenTelemetry users who are authoring metrics or metadata.
+
+All of this functionality is additive to OpenTelemetry’s existing tracing support, and both signal types share the same metadata and semantic conventions. As of this announcement, the Java, JS, .Net, and Python SDKs have issued release candidates for OpenTelemetry’s metric capabilities, and more languages will be issuing metrics release candidates soon. Each of these releases will be followed by general availability after we receive feedback from users.
+
+## Getting Started
+If you’re already using a mix of the OpenTelemetry APIs, SDKs, language agents, or Collector, then you can access GA metrics functionality by updating your OpenTelemetry artifacts to their latest versions. You can find documentation and getting started guides for each artifact’s metrics capabilities within [the official OpenTelemetry documentation](../../../docs) and examples and supplementary documentation is often available within each artifact’s corresponding GitHub repository.
+
+## What’s Next for OpenTelemetry
+WDistributed traces and logs were the two halves of OpenTelemetry’s core promise when we announced it at Kubecon EU in 2019. With the general availability of metrics, we have produced the capabilities that we originally set out to create, meaning that we can shift our focus to further investing into the robustness and ease of use of each component, the number of data sources that OpenTelemetry can capture telemetry from (either through the OpenTelemetry APIs, OTLP, or otherwise), and new capabilities and signal types.
```

Comment:
```suggestion
Distributed traces and logs were the two halves of OpenTelemetry’s core promise when we announced it at Kubecon EU in 2019. With the general availability of metrics, we have produced the capabilities that we originally set out to create, meaning that we can shift our focus to further investing into the robustness and ease of use of each component, the number of data sources that OpenTelemetry can capture telemetry from (either through the OpenTelemetry APIs, OTLP, or otherwise), and new capabilities and signal types.
```